manual/src/refman/extensions/extensiblevariants.etex

```diff
@@ -67,6 +67,27 @@ type Expr.attr += Str = Expr.Str
 let now_works = Str "foo";;
 \end{caml_example}
 
+Extensible variant types can be reexported in another module,
+for instance to make the type appear in the most natural module
+for the end user of a library.
+\begin{caml_example*}{verbatim}
+module Expr2 : sig
+  type t = ..
```

Comment:
Sorry for not catching this earlier but this is a contradiction between the code and the text: you are not-reexporting the type `Expr.attr`. I would suggest to not abstract the type and remove the following paragraph.

Reply:
I first tried to rephrase. I think the example is what people needs  (I generalized this from one example I used ;-).

Your comments made me think that a lexicon on the vocabulary about types and values would be real nice. Maybe a short introduction in the documentation about functionnal programming and types, using the vocabulary used later in the documentation ?

Reply:
I disagree, examples should be as focused as possible. Doing two things at the same time increase the risk of confusing the reader.

In other words, if we document how to re-export extension types, we should give an example doing only a reexport. (It is not like it is a major inference step to learn how to mix abstraction and re-export.)

Concerning your second point, I am not convinced that a lexicon would be good fit with the style of the manual.
And it would rather belong to an annex rather than an introduction. And anyway, such a lexicon would not belong to this PR.